spec/src/main/asciidoc/Connectors.adoc

```diff
@@ -18848,11 +18851,11 @@ archive.
 === @AdministeredObjectDefinitions
 
 The _AdministeredObjectDefinition_ annotation
-is a resource definition annotation that is used to define an
-administered object and have it registered in JNDI. Since repeated
-annotations are not allowed, the _AdministeredObjectDefinitions_
+is an optional resource definition annotation that is used to define an
```

Comment:
Did you mean to add the term 'optional' to AdministeredObjectDefinition here?  It's optional in the sense that users get to decide which, if any, they wish to use, but shouldn't be optional for the application server to process/implement.

Reply:
Yeah that wording is loose I will amend. I was writing in terms for a Jakarta EE developer rather than the implementer.

Reply:
OK now removed this wording